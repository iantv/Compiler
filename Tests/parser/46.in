void main(){
	2;
	printf("%d", 1);
}