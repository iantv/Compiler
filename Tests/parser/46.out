main: function
	{
		expression:
			2
		printf:
			
				expression:
					%d
				expression:
					1
	}
returns
	void
